packages/lab/src/checkbox/CheckboxBase.css

```diff
@@ -45,35 +15,6 @@
   cursor: var(--uitkCheckbox-cursor-disabled, var(--uitk-selectable-cursor-disabled));
```

Comment:
this class doesnt get applied on disabled

Reply:
its now redundant